packages/sample/package.json

```diff
@@ -3,20 +3,23 @@
     "descripton": "sample for testing Rev Hub code",
     "version": "1.0.0",
     "main": "dist/main.js",
-    "bin": "revhub",
+    "bin": "dist/main.js",
     "private": true,
     "type": "module",
     "dependencies": {
         "@rev-robotics/distance-sensor": "^1.0.0",
         "@rev-robotics/expansion-hub": "^1.0.0",
         "commander": "^10.0.1"
     },
+    "devDependencies": {
+        "typescript": "^5.0.2"
+    },
     "scripts": {
         "build": "tsc",
         "motor": "node dist/main.js motor power 1 0.60",
         "analog": "node dist/main.js analog 0",
-        "battery": "node dist/main.js battery --continuous",
-        "voltage": "node dist/main.js voltage --continuous",
+        "voltage": "node dist/main.js battery voltage --continuous",
+        "current": "node dist/main.js battery current --continuous",
```

Comment:
Do we still want to keep these scripts? We don't have full coverage of the sample via these scripts, so I would be ok with removing them.

Reply:
Fair point, I'll just do that